server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/AppCenterNotification.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Arcadia.Assistant.AppCenterBuilds
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Employees.Contracts;
+
+    using Microsoft.Extensions.Logging;
+
+    using Notifications.Contracts;
+    using Notifications.Contracts.Models;
+
+    public class AppCenterNotification : IAppCenterNotification
+    {
+        private readonly string clientName;
+        private readonly ILogger logger;
+        private readonly INotifications notifications;
+
+        public AppCenterNotification(string clientName, INotifications notifications, ILogger logger)
+        {
+            this.clientName = clientName;
+            this.notifications = notifications;
+            this.logger = logger;
+        }
+
+        async Task IAppCenterNotification.Notify(string buildVersion, string mobileType, CancellationToken cancellationToken)
+        {
+            this.logger.LogDebug($"Send app center build notification about build version {buildVersion} for '{mobileType}' platform.");
```

Comment:
do not use string interpolation in logs, you break aggregations. use `LogDebug("Build is {BuildVersion}", buildVersion) instead.

Reply:
That applies to all log messages. I might have done the same mistake somewhere :)

Reply:
All logging cases fixed (except some "old" services)